lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -678,7 +682,7 @@ import qualified Cardano.Wallet.Primitive.Types.TokenBundle as TokenBundle
 import qualified Cardano.Wallet.Primitive.Types.TokenMap as TokenMap
 import qualified Cardano.Wallet.Primitive.Types.Tx as W
 import qualified Cardano.Wallet.Primitive.Types.UTxO as UTxO
-import qualified Cardano.Wallet.Primitive.Types.UTxOIndex as UTxOIndex
+import qualified Cardano.Wallet.Primitive.Types.UTxOIndex.Internal as UTxOIndex
```

Comment:
The `.Internal` modules are usually named such because they aren't meant to be used directly. I don't see a reason to depend on it.